pkg/block/fetcher.go

```diff
@@ -533,6 +534,23 @@ func contains(s1 []ulid.ULID, s2 []ulid.ULID) bool {
 	return true
 }
 
+type ReplicaLabelsFilter struct {
+	ReplicaLabels []string
+}
+
+func (f *ReplicaLabelsFilter) Filter(metas map[ulid.ULID]*metadata.Meta, synced GaugeLabeled, view bool) {
```

Comment:
```suggestion
func (f *ReplicaLabelsFilter) PostFetch<SOMETHING>(metas map[ulid.ULID]*metadata.Meta, synced GaugeLabeled, view bool) {
```

![](https://media3.giphy.com/media/EtMlS5JeC6FBS/giphy.gif)